src/k8s/cmd/k8s/k8s_get.go

```diff
@@ -116,6 +115,8 @@ func newGetCmd(env cmdutil.ExecutionEnvironment) *cobra.Command {
 				output = config.LoadBalancer.GetBGPPeerPort()
 			case fmt.Sprintf("%s.bgp-peer-asn", features.LoadBalancer):
 				output = config.LoadBalancer.GetBGPPeerASN()
+			case "cloud-provider":
+				output = config.GetCloudProvider()
```

Comment:
what happens if it's unset?  I guess that's okay -- it doesn't panic by de referencing nil or something?

Reply:
util.Deref(c.CloudProvider) in the k8s-snap-api dereferences in a safe way. If it's not set, it won't print anything.